doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+
+3.  Connect the internal Gateway to the back-end service.
+
+    For some services, such as `proxy` or `amqp.proxy`, you configure the [connect](../admin-reference/r_configure_gateway_service.md#connect) element to connect to the back-end service used by the Gateway. Other services, such as JMS, use the [properties](../admin-reference/r_conf_jms.md) element to connect the internal Gateway to a JMS-compliant message broker.
+
+    The connect element can specify either the hostname or the IP address, and port of the back-end service:
+
+    `                <connect>tcp://127.0.0.1:3101</connect>          `
+
+    When using the JMS service, the properties element names the ConnectionFactory, queue and topic names, and the URI for the message broker. See the [properties](../admin-reference/r_conf_jms.md#env_prop_stompjms) element for an example.
+
+4.  Add the [accept-options](../admin-reference/r_configure_gateway_service.md#accept-options-and-connect-options) element:
+
+    ``` xml:
+    <accept-options>
+      <http.transport>socks://${dmz.backplane.hostname}:1080</http.transport>
+      <socks.mode>reverse</socks.mode>
+      <socks.retry.maximum.interval>1 second</socks.retry.maximum.interval>
+      <socks.transport>
+        wsn+ssl://${dmz.backplane.hostname}:1080/shield
+      </socks.transport>
+      <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>
+    </accept-options>
+    ```
+
+    The [accept-options](../admin-reference/r_configure_gateway_service.md#accept-options-and-connect-options) settings are the most interesting part of the configuration because the settings change the nature of the service and affect the connections for the App1 service:
+
+    -   Set [http.transport](../admin-reference/r_configure_gateway_service.md#protocoltransport) to use the SOCKS protocol:
+    
+         `                     <http.transport>${dmz.backplane.hostname}:1080</http.transport>                 `
+
+        Enterprise Shield™ uses the SOCKS protocol to establish connections in reverse, connecting from the internal Gateway to the DMZ Gateway. By default, the SOCKS connection is still a forward connection, from the DMZ Gateway to the internal Gateway. The next accept-options element, `socks.mode`, is the setting you use to make the connection go in reverse.
+
+        Port 1080 is the convention for SOCKS connections, but you can use any port.
+
+        **Note:** The settings for `http.transport` and `socks.mode` must match on both the DMZ Gateway and the internal Gateway.
```

Comment:
How about a second note saying something like:

> **Note:** This example uses a property (`${dmz.backplane.hostname}`) to specify the hostname, because the hostname is referenced more than once in the configuration.

Or is that bloating things too much?